src/Chip/Chip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import "./chip.scss";
+
+export type ChipProps = React.PropsWithChildren<{
+    id?: string;
+    className?: string;
+    onClose: (e: React.MouseEvent<HTMLDivElement>) => void;
+}>;
+
+const Chip: React.FC<ChipProps> = (props: ChipProps) => {
```

Comment:
as Nuru mentioned before, React.FC will be removed in latest react component, probably you should change to `React.FunctionComponent`

Reply:
It's not, `SFC` and `StatelessComponent` are deprecated:
```typescript
/**
  * @deprecated as of recent React versions, function components can no
  * longer be considered 'stateless'. Please use `FunctionComponent` instead.
  *
  * @see [React Hooks](https://reactjs.org/docs/hooks-intro.html)
  */
type SFC<P = {}> = FunctionComponent<P>;

/**
  * @deprecated as of recent React versions, function components can no
  * longer be considered 'stateless'. Please use `FunctionComponent` instead.
  *
  * @see [React Hooks](https://reactjs.org/docs/hooks-intro.html)
  */
type StatelessComponent<P = {}> = FunctionComponent<P>;

type FC<P = {}> = FunctionComponent<P>;

interface FunctionComponent<P = {}> {
    (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
    propTypes?: WeakValidationMap<P>;
    contextTypes?: ValidationMap<any>;
    defaultProps?: Partial<P>;
    displayName?: string;
}
```

Reply:
@yousifalraheem regardless, i think you should ditch .SF . Kp shared this link with me and i think we should reconsider .SF https://github.com/facebook/create-react-app/pull/8177 . So FunctionComponent is more favorable .